flipperdevices/flipperzero-firmware
# [FL-2511] Updater: fixed long update descriptions overlapping UI elements

# What's new

- [FL-2511] Updater: fixed long update descriptions overlapping UI elements

# Verification 

- Build & install
- Create a firmware update package with description "gornek_2507_amiibo_emulation_fix-28042022-459a2134 / gornek/2507_amiibo_emulation_fix"
- Open update package on Flipper and check that text is clipped 

# Checklist (For Reviewer)



[FL-2511]: https://flipperzero.atlassian.net/browse/FL-2511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
